client/src/components/Forms/ModalForm.tsx

```diff
@@ -22,7 +22,12 @@ export function ModalForm(props: Props) {
   if (props.data == null) {
     return null;
   }
-  const initialValues = props.getInitialValues ? props.getInitialValues?.(props.data) : props.data;
+
+  const formValues = props.getInitialValues ? props.getInitialValues?.(props.data) : props.data;
+
+  form.resetFields();
```

Comment:
i don't think it's ok to reset on every component render globally

Reply:
Yep, we should do it only on new data come, fixed